xapian-letor/include/xapian-letor/feature.h

```diff
@@ -190,6 +190,14 @@ class XAPIAN_VISIBILITY_DEFAULT CollTfCollLenFeature : public Feature {
     std::string name() const;
 };
 
+// This feature consists of a constant value
+class XAPIAN_VISIBILITY_DEFAULT ConstantFeature : public Feature {
+  public:
+    ConstantFeature() {}
```

Comment:
The commit message says `Add ConstantFeature(value=1.0)` but you're actually only implementing `ConstantFeature()` with the value hard-coded as `1.0`.

I don't know if it's useful to be able to specify the constant value for a bias feature (so I'm not sure if it's the commit message or the code that's wrong here) - I just know that the commit message really ought to match what the commit actually implements!

Reply:
fixed in a fixup commit.